src/main/java/com/atlassian/db/replica/internal/aurora/AuroraConnectionSupplier.java

```diff
@@ -0,0 +1,9 @@
+package com.atlassian.db.replica.internal.aurora;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+@FunctionalInterface
+public interface AuroraConnectionSupplier {
```

Comment:
Shouldn't this be part of SPI?

Reply:
renamed the interface to fit better and moved it to the spi package